docs/serving/samples/hello-world/helloworld-csharp/README.md

```diff
@@ -181,7 +184,7 @@ folder) you're ready to build and deploy the sample app.
    the URL below with the URL returned in the previous command.
 
    ```shell
-   curl http://helloworld-csharp.default.1.2.3.4.xip.io
+   curl -H "Host: helloworld-csharp.default.1.2.3.4.xip.io" http://localhost
```

Comment:
This shouldn't necessarily point to localhost -- it depends on where your loadbalancer lives. I'd suggest instead editing `config-domain` to have xip.io reflect the LoadBalancer or NodePort's address.